### PR TITLE
Docs 6872 pm2.x delete certs m

### PR DESCRIPTION
**Pull request by mcherem**
Delete Certificates, Endpoints (also create) from Partners, Host

### CODE REVIEW

**Review by amitfernandes — APPROVED** [review]
Approving "delete certificates" change

File: modules/ROOT/pages/configure-host.adoc [hunk 1]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
 
 . Click *New* within the *Certificates* section to begin to upload any needed digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
 
-. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload the chosen digital certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
+. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]:
```suggestion
. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. 
See xref:Certificates.adoc[Certificates] for more information about configuring certificates in Partner Manager.
```

File: modules/ROOT/pages/configure-host.adoc [hunk 2]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
++
+NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: Does this really belong at this location? Is the user deleting or planning to delete a certificate at this point?

File: modules/ROOT/pages/configure-host.adoc [hunk 3]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
++
+. Click *New* from within _Endpoints: Receive from partners_ or _Endpoints: Targets at <partner name>_ for the Host, and select your protocol type.
 
-. (Optional) Click *Edit* within the *Payload storage* section to change any of the information that was completed when the payload storage was set up. For example, URL location, timeout settings, and acknowledgments.
+. Set up your endpoint based on the protocol type you selected.
++
+For more information on your protocol type refer to:
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]:
```suggestion
For more information about your protocol type, refer to:
```

File: modules/ROOT/pages/configure-host.adoc [hunk 4]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
+
+* xref:endpoint-sftp-receive-target.adoc[SFTP Receive Endpoint Settings]
+
+* xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
++
+. Save when you are done.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: How? Do they click something?

File: modules/ROOT/pages/configure-host.adoc [hunk 5]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
+
+. If you need to change any of the information that was completed when the payload storage was set up, click *Edit* within the *Payload storage* section.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: Could this be "If you need to make changes later, click...."

File: modules/ROOT/pages/configure-host.adoc [hunk 6]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
+
+For example, URL location, timeout settings, and acknowledgments.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: This is a fragment. Please incorporate it into the sentence above or make it complete in itself.

File: modules/ROOT/pages/configure-host.adoc [hunk 7]
```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
...
 
 Refer to xref:x12-identity-settings.adoc[Identifiers] for information on deleting an identifier.
 
+Refer to xref:endpoints.adoc[Endpoint Types and Protocols] for information on deleting an endpoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-host.adoc]: GLOBALLY change "information on" to "information about" in all writing.
I will stop marking it from here on.

File: modules/ROOT/pages/configure-partner.adoc [hunk 8]
```diff
@@ -32,6 +32,20 @@ For example, depending on your setup, select one of the following: AS2 ID, X12-I
 
 See xref:x12-identity-settings.adoc[Identifiers] for more info on X12 Identity settings.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
See xref:x12-identity-settings.adoc[Identifiers] for more information about X12 Identity settings.
```

File: modules/ROOT/pages/configure-partner.adoc [hunk 9]
```diff
@@ -32,6 +32,20 @@ For example, depending on your setup, select one of the following: AS2 ID, X12-I
...
 
+. Click *Create partner*.
+
+. If you want to add an endpoint now, click *New* under _Endpoints: Send to <partner name>_.
+
+. Select the Protocol-Type of the Endpoint for which you are entering settings.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc:37]:
```suggestion
. Select the protocol type of the endpoint for which you are entering settings.
```

File: modules/ROOT/pages/configure-partner.adoc [hunk 10]
```diff
@@ -62,6 +76,10 @@ See xref:x12-identity-settings.adoc[Identifiers] for more info on X12 settings.
 
 . Click *New* within the *Certificates* section to upload any needed digital certificates. This is your partner's digital certificate (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
 
++
+NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc:78]: Again evaluate whether your user really needs to think about deletion at this point.

File: modules/ROOT/pages/configure-partner.adoc [hunk 11]
```diff
@@ -62,6 +76,10 @@ See xref:x12-identity-settings.adoc[Identifiers] for more info on X12 settings.
...
++
+
 * Click *Choose file* to select and upload the certificate.
 
 * Click *Upload certificate* (typically a .PEM file provided by your trading partner) to upload the chosen digital certificate file.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: .PEM file is formatted as syntax earlier.
I think neither is correct. I suggest "PEM file" or ".pem file," no formatting.
Please search for and evaluate globally.

File: modules/ROOT/pages/configure-partner.adoc [hunk 12]
```diff
@@ -122,6 +140,19 @@ You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identif
 
 . If you need a replacement, add it now by clicking *New*.
 
+== Delete Endpoints
+
+You can delete any endpoints that are not used in a message flow configuration.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
You can delete any endpoints that are not used in a message flow configuration:
```

File: modules/ROOT/pages/configure-partner.adoc [hunk 13]
```diff
@@ -122,6 +140,19 @@ You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identif
...
+
+. From within your Anypoint Partner Manager Sandbox or Production environment, select <host partner name> from the upper left.
+
+. From _Endpoints: Send to <partner name>_ search for the endpoint you want to delete by Name, Protocol, URL, or Description.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
. From _Endpoints: Send to <partner name>_ , search for the endpoint you want to delete by name, protocol, URL, or description.
```

File: modules/ROOT/pages/configure-partner.adoc [hunk 14]
```diff
@@ -122,6 +140,19 @@ You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identif
...
++
+If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]: Earlier text mentions the 0 in the endpoint column?

File: modules/ROOT/pages/configure-partner.adoc [hunk 15]
```diff
@@ -122,6 +140,19 @@ You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identif
...
++
+. Hover over the endpoint name with your mouse until you see the garbage can icon, click it and click *Delete*.
```
Comment (kbostrom) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
. Hover over the endpoint name until you see the garbage can icon, click the icon, and then click *Delete*.
```

**Review by kbostrom — APPROVED** [review]
Done.

File: modules/ROOT/pages/configure-host.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 5 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 7 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

File: modules/ROOT/pages/configure-partner.adoc (hunk 8 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: Fixed.

File: modules/ROOT/pages/configure-partner.adoc (hunk 9 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc:37]: This edit changes the meaning.

File: modules/ROOT/pages/configure-partner.adoc (hunk 10 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc:78]: Keep.

File: modules/ROOT/pages/configure-partner.adoc (hunk 12 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: Fixed.

File: modules/ROOT/pages/configure-partner.adoc (hunk 11 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: fixed to '<certificate name>.pem'

File: modules/ROOT/pages/configure-partner.adoc (hunk 13 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: Fixed.

File: modules/ROOT/pages/configure-partner.adoc (hunk 14 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: Yes, fixed.

File: modules/ROOT/pages/configure-partner.adoc (hunk 15 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-partner.adoc]: Fixed.

File: modules/ROOT/pages/configure-host.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/configure-host.adoc]: Fixed.

**Pull request merged**